deploy/kubernetes/charts/capact/charts/engine/values.yaml

```diff
@@ -60,67 +60,57 @@ affinity: {}
-            - implementationConstraints: # prefer Implementation for Kubernetes
-                requires:
-                  - path: "cap.core.type.platform.kubernetes"
-                    # any revision
-            - implementationConstraints: {} # fallback to any Implementation
-
-testSetup:
-  enabled: false
-  globalPolicyRules:
-    - interface:
-        path: "cap.interface.capactio.capact.validation.action.passing"
-      oneOf:
-        - implementationConstraints:
-            requires:
-              - path: "cap.type.capactio.capact.validation.single-key"
-            attributes:
-              - path: "cap.attribute.capactio.capact.validation.policy.most-preferred"
-          inject:
-            requiredTypeInstances: [] # DO NOT MODIFY this line, as it is found and replaced during integration tests
-        - implementationConstraints:
-            path: cap.implementation.capactio.capact.validation.action.passing-a
+#  interface:
+#    rules:
+#      - interface:
+#          # Rules for exact path and revision (looked up in first place)
+#          path: "cap.interface.database.postgresql.install"
+#          revision: "0.1.0"
+#        oneOf:
+#          - implementationConstraints:
+#              requires:
+#                - path: "cap.type.gcp.auth.service-account"
+#                  # any revision
+#              attributes:
+#                - path: "cap.attribute.cloud.provider.gcp"
+#                  # any revision
+#            inject:
+#             typeInstances:
+#               - id: "{uuid}"
+#                 typeRef:
+#                   path: "cap.type.gcp.auth.service-account"
+#                   revision: "0.1.0"
+#          - implementationConstraints:
+#              attributes:
+#               - path: cap.attribute.cloud.provider.aws
+#                 # any revision
+#          - implementationConstraints:
+#              path: cap.implementation.bitnami.postgresql.install
+#      - interface:
+#          # Rules for exact path in any revision (looked up in second place)
+#          path: "cap.interface.database.postgresql.install"
+#        oneOf:
+#          - implementationConstraints:
+#              requires:
+#                - path: "cap.type.gcp.auth.service-account"
+#                  # any revision
+#              attributes:
+#                - path: "cap.attribute.cloud.provider.gcp"
+#                  # any revision
+  interface:
+    rules:
+      - interface:
+          # any other Interface (looked up in third place, if there is no rule for `path` and `revision` or `path`)
+          path: "cap.*"
+        oneOf:
+          - implementationConstraints: # prefer Implementation for Kubernetes
+              requires:
+                - path: "cap.core.type.platform.kubernetes"
+                  # any revision
+          - implementationConstraints: {} # fallback to any Implementation
+
+## Currently, not used but setting this filed is supported by all scripts and CLI.
```

Comment:
If we comment out this value from the Chart, would it work:
```
// from hack/lib
CAPACT_OVERRIDES+=",engine.testSetup.enabled=${USE_TEST_SETUP}"
```
? 
I didn't check it but I am wondering if it is ok to set up a value that does not exist in the Helm chart.

Reply:
in `values.yaml` file you store only default that you want to have. Still others may specify it and chart template may use it.
![Screen Shot 2022-02-04 at 09 17 44](https://user-images.githubusercontent.com/17568639/152495073-b6196098-4d6d-439e-b52d-49539ed891ab.png)
 
 
 As we don't have https://helm.sh/docs/topics/charts/#schema-files it won't report any error.